lib/src/runner/loader.dart

```diff
@@ -211,6 +211,10 @@ class Loader {
       return;
     }
 
+    if (_config.suiteDefaults.excludeTags.evaluate(suiteConfig.metadata.tags)) {
+      return;
+    }
```

Comment:
I'm guessing the return can't fit on a single line under 80 characters? Hard to tell in the Github review UI.

Reply:
Yep.